logging/bin/deploy_elasticsearch_open.sh

```diff
@@ -356,7 +356,7 @@ fi
 # TO DO: Convert to curl command to detect ES is up?
 # hitting https:/host:port -u adminuser:adminpwd --insecure 
 # returns "Open Distro Security not initialized." and 503 when up
-log_verbose "Waiting [2] minute to allow Elasticsearch to initialize [$(date)]"
+log_verbose "Waiting up to [5] minute to allow Elasticsearch to initialize [$(date)]"
 sleep 120s
```

Comment:
This says '[5]' minutes, but the sleep is 120s

Reply:
Corrected.